instrumentation/cassandra/cassandra-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v4_0/CassandraSingletons.java

```diff
@@ -37,7 +37,7 @@ public final class CassandraSingletons {
                     .build())
             .addAttributesExtractor(
                 NetworkAttributesExtractor.create(new CassandraNetworkAttributesGetter()))
-            .addAttributesExtractor(new CassandraAttributesExtractor())
+            .addAttributesExtractor(new CassandraServerAttributesExtractor())
```

Comment:
I think we need both here

Reply:
line 39 and 40. I renamed CassandraAttributesExtractor to CassandraServerAttributesExtractor.

Reply:
we keep both network and server extractors